src/app/exam/ExamProblems.js

```diff
@@ -48,12 +31,14 @@ const ExamProblems = withRouter(({ history, match }) => {
     };
 
     const addProblem = (problemId, scorePercentage, submissionQuota) => {
-        examService.addExamQuestion({
+        const addExamPromise = examService.addExamQuestion({
```

Comment:
please make sure every term you wrote is accurate.
 rename `addExamPromise` to `addQuestionPromise`

Reply:
I'm sorry about this